adam-core/src/main/scala/org/bdgenomics/adam/converters/AlignmentRecordConverter.scala

```diff
@@ -21,7 +21,12 @@ import htsjdk.samtools.{ SAMFileHeader, SAMRecord }
 import org.bdgenomics.adam.instrumentation.Timers._
 import org.bdgenomics.adam.models._
 import org.bdgenomics.adam.rich.RichAlignmentRecord
-import org.bdgenomics.formats.avro.AlignmentRecord
+import org.bdgenomics.formats.avro.{
+  AlignmentRecord,
+  Fragment,
+  Sequence
+}
+import scala.collection.JavaConversions._
 
 class AlignmentRecordConverter extends Serializable {
 
```

Comment:
The comment for `convertToFastq` should look like, e.g.

```
   * {{{
   * @readName
   * sequence
   * +<optional readname>
   * ASCII quality scores
   * }}}
```
in order to keep Scaladoc from being confused and thinking that @readName is an annotation.